### DISCUSSION TITLE
Welcome to vagrant-vbguest Discussions!

### DISCUSSION THREAD

**Discussion by fnordfish** [dotless-de/vagrant-vbguest#397] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * Please keep the [code of conduct](https://github.com/dotless-de/vagrant-vbguest/blob/main/CODE_OF_CONDUCT.md) in mind.

**Comment by Eddie-Muffin** [comment]
please kindly help me solve this problem. I have been on it for days now. I do not know what to day. Thank you


$ vagrant plugin install vagrant-vbguest
Installing the 'vagrant-vbguest' plugin. This can take a few minutes...
ERROR:  SSL verification error at depth 3: unable to get local issuer certificate (20)
ERROR:  You must add /C=US/O=Starfield Technologies, Inc./OU=Starfield Class 2 Certification Authority to your local trusted store
Vagrant failed to load a configured plugin source. This can be caused
by a variety of issues including: transient connectivity issues, proxy
filtering rejecting access to a configured plugin source, or a configured
plugin source not responding correctly. Please review the error message
below to help resolve the issue:

  SSL_connect SYSCALL returned=5 errno=0 peeraddr=54.192.95.124:443 state=error: certificate verify failed

**Reply by fnordfish** [reply]
The Vagrant Plugin System, which uses [rubygems](https://rubygems.org), could not establish a secure connection to the rubygems server. This can have numerous reasons, like man-in-the-middle attacks, misconfigured or old TLS certificate chains. Without knowing anything about your host operating system, it's version and the version of vagrant you are using, it's very hard to tell.

However, this is no a plugin specific issue. You'll most likely have the same issue installing any vagrant plugin. 
Please have a look here: https://discuss.hashicorp.com/t/vagrant-2-3-5-unable-to-install-plugins/53916
